### PR TITLE
Adding PyTest workflow

### LINKED ISSUE #32: Adding pytest YAML file

### PR DESCRIPTION
**Pull request by efraimvis**
Closes #32 
- Added pytest.yml, a workflow that will run the tests we provide for the Django code in our PRs
- CAUTION: please notice that some files, such as "models.py" in "TastiesApp", required imports to be commented out in order to pass flake8 tests. When we actually write code in said files, we of course need to uncomment said imports.

### CODE REVIEW

File: TastiesApp/apps.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from django.apps import AppConfig
 
 
-class WebpageConfig(AppConfig):
+class TastiesappConfig(AppConfig):
```
Comment (GilShechter) [inline TastiesApp/apps.py:4]: Please fix the name to 'Tasties**A**ppConfig' (Python convention).

Comment (efraimvis) [reply TastiesApp/apps.py]: Thanks. Done.

**Review by WaseemAbuSobe — APPROVED** [review]
LGTM

**Review by GilShechter — DISMISSED** [review]
LGTM

**Review by GalOved — DISMISSED** [review]
looking good for me

**Review by WaseemAbuSobe — DISMISSED** [review]
LGTM

**Review by ohadrevach — APPROVED** [review]
LGTM

**Review by natifridman — CHANGES_REQUESTED** [review]
Please change the PR title and description to explain what is this PR for.
Do you need the `caution` as part of the PR description?

Also, please follow [pep8](https://peps.python.org/pep-0008/#package-and-module-names) for module names and class names standard.

**Comment by efraimvis** [comment]
> Please change the PR title and description to explain what is this PR for. Do you need the `caution` as part of the PR description?

It seemed appropriate to mention because it's very easy to miss import statements, and since this was our first PR that actually had to pass the flake8 workflow, it might not be obvious to everyone that flake8 will demand this comment-out.
> 
> Also, please follow [pep8](https://peps.python.org/pep-0008/#package-and-module-names) for module names and class names standard.

Could you please elaborate, before I try to make any changes, where in my case I deviated from the standard?

**Pull request closed without merging**